src/IssueViz/Selection/AnalysisIssueNavigation.cs

```diff
@@ -22,36 +22,39 @@
 using System.Collections.Generic;
 using System.ComponentModel.Composition;
 using System.Linq;
+using SonarLint.VisualStudio.IssueVisualization.Editor;
 using SonarLint.VisualStudio.IssueVisualization.Models;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Selection
 {
     internal interface IAnalysisIssueNavigation
```

Comment:
Speaking of naming, I think the name of this service is confusing when we also have ILocationNavigator.
Maybe it should be something like ISelectedLocationCycler? (and in the future, ISelectedFlowCycler, with one service SelectionCycler?)

Reply:
`ILocationNavigationCommands`, `IIssueStepNavigator`, `IIssueFlowStepNavigator`...